webauthn/webauthn.py

```diff
@@ -1167,6 +1176,21 @@ def _load_cose_public_key(key_bytes):
 
         return alg, RSAPublicNumbers(e,
                                      n).public_key(backend=default_backend())
+    elif alg == COSE_ALG_EdDSA:
+        if ed25519 is None:
+            raise COSEKeyException('Unsupported algorithm.')
```

Comment:
Added this check. Should the text be something on the lines of "Unable to import ed25519' or this good enough?

Reply:
"Unable to import ed25519, check cryptography version (min 2.6 needed)"  - or something that says this seems like an actual useful error message.

Reply:
I like the more descriptive message